pkg/router/route.go

```diff
@@ -22,11 +21,12 @@ func ConfigureEcho(allRoutes bool) *echo.Echo {
 	echoLogger := lecho.From(log.Logger,
 		lecho.WithTimestamp(),
 		lecho.WithCaller(),
-		lecho.WithLevel(echo_log.INFO),
 	)
+
 	e.Use(echo_middleware.RequestIDWithConfig(echo_middleware.RequestIDConfig{
 		TargetHeader: config.HeaderRequestId,
 	}))
+	e.Use(middleware.AddRequestId)
```

Comment:
Lecho tries to get the request ID from the header, not the context. So even if the AddRequestId middleware has added it to the context, lecho can't find it.

The echo middleware generates a request ID, and sets it on the response header, so then that request ID gets used.

Reply:
good catch.  I updated it to set it properly on the request/response, as well as not use the echo middleware since it doesn't set it on the context